### DISCUSSION TITLE
Potential audiences for ASG white papers

### DISCUSSION THREAD

**Discussion by RaoOfPhysics** [alan-turing-institute/asg-research-communications#80] [General]
The potential audiences and stakeholders for the white papers (per the job description) include

> “policy makers in national and local governments, domain expert practitioners such as clinicians or engineers, and members of the public”.

Each of the papers may have a slightly different primary audience. This will have implications for the style, tone, message and focus of recommendations for each paper.

e.g. [potentially but not exclusively]:

1. **Environment and Sustainability** &ndash; [including suggestions made during the kick-off]
    * National Environment Research Council [NERC] and national laboratories (BAS, NOC, CEH etc.)
    * Met Office
    * Universities
    * National government
    * …
2. **Ecosystem of Digital Twins**
    * Industry (e.g. Rolls Royce, Lloyds Registry)
    * Regional authorities (city councils, counties, bodies like Transport for the North)
    * Hospitals administrators
    * National government
    * …
3. **Shocks and Resilience**
    * Industry
    * National government
    * NHS (in all constituent countries)
    * …

[Also, see #78.]

**Comment by ZaynabIsmail** [comment]
The link for this issue has been added to the 'Our target audience' tab of the [impact spreadsheet ](https://thealanturininstitute.sharepoint.com/:x:/s/rid/ESpjyZFOm3tLgeODnYxwyh0BpjrPwOv6RxgttAShCTk7TA?e=sFAMSa).

For the second and third papers the 'Our target audience' tab will be updated once the audience becomes more defined.

**Comment by ZaynabIsmail** [comment]
[Other potential audiences](https://thealanturininstitute.sharepoint.com/:x:/s/rid/EalTmrNYHQpBm6xiGk5kumABuwp6SxHDw8dDRn2Ef9ABFA?e=p9aynA) for the E&S paper for @RaoOfPhysics to review.
